run.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+/bin/fake-gcs-server \
+  -backend "${BACKEND:-filesystem}" \
+  -cert-location "${CERT_LOCATION}" \
+  -cors-headers "${CORS_HEADERS}" \
```

Comment:
Ugh those won't really work when those environment variables aren't set.

I'll close this PR for now, but if anyone is willing to pick it back up, I think it would be better to have environment variables handled directly in the code, to avoid having a shell script with too many conditionals.